### PR TITLE
update to latest reality-data-client pkg

### PR DESCRIPTION
**Pull request by aruniverse**

### CODE REVIEW

**Review by hl662 — APPROVED** [review]

**Review by pmconne — APPROVED** [review]
Fixes integration test failure?

**Comment by aruniverse** [comment]
> Fixes integration test failure?

Not necessarily fixed by this PR. But they have been fixed.
It was a permission issue, where a certain realitydata permission was removed from the integration test user

**Comment by aruniverse** [comment]
@mcote01 tests are failing again now? Did you change the permissions again?

**Comment by mcote01** [comment]
@aruniverse I set permissions again to the test user, should wait a few minutes for permission caches to clear and then try running tests again

**Comment by aruniverse** [comment]
> @aruniverse I set permissions again to the test user, should wait a few minutes for service caches to clear and then try running tests again

It is still failing. Any idea what's going on?

I will skip this test if it continues to fsil today because it's been blocking every pr since mid last week

**Comment by pmconne** [comment]
> I will skip this test if it continues to fsil today

Please do. We can force-merge normal PRs to bypass it but it's going to block the publication of new addons, which is expecting to happen today.

File: full-stack-tests/core/src/frontend/standalone/RealityDataAccess.test.ts [hunk 1]
```diff
@@ -243,7 +243,7 @@ describe("RealityDataAccess (#integration)", () => {
     }
   });
 
-  it("should get RealityDataSource for reality data attachment in iModel", async () => {
+  it.skip("should get RealityDataSource for reality data attachment in iModel", async () => {
```
Comment (aruniverse) [inline full-stack-tests/core/src/frontend/standalone/RealityDataAccess.test.ts:246]: Temporarily skip tests until permissions are sorted out.

**Pull request merged**